docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to use `earthdata-varinfo` to publish UMM-Var entries to CMR"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Version 2.0.0"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview:"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This version of `earthdata-varinfo` adds functionality to publish records to CMR, along with a single overarching function that wraps the search, download and publication functionality into a single function for the convenience of the end-user. Additionally, the function signatures for `cmr_search.get_granules` and `cmr_search.download_granule` have been updated to accept the full Authorization header, instead of a bearer token, so that they are also compatible with LaunchPad tokens."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up your environment to run this notebook"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "pip install earthdata-varinfo\n",
+    "```\n",
+    "\n",
+    "If this doesn't work, alternatively you can clone the git repository, and install the package in editable mode:\n",
+    "\n",
+    "```\n",
+    "git clone https://github.com/nasa/earthdata-varinfo\n",
+    "cd earthdata-varinfo\n",
+    "pip install -e .\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other notebook requirements:\n",
+    "\n",
+    "Required packages for `earthdata-varinfo` should be installed as dependencies of PyPI. For local development, without a standard pip installation, third party requirements can be installed from the following files:\n",
+    "\n",
+    "```\n",
+    "pip install -r requirements.txt -r dev-requirements.txt\n",
+    "pip install notebook\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Current output format\n",
+    "Currently `earthdata-varinfo` produces Python classes that can be used within Python scripts, Jupyter notebooks or a Python Read-Eval-Print Loop (REPL).\n",
+    "\n",
+    "`earthdata-varinfo` is also able to create UMM-Var compliant JSON records and publish these records to NASA's Common Metadata Repository (CMR)."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example usage:\n",
```

Comment:
These examples work for folks with the correct privileges for the EEDTEST CMR provider in UAT. I think there's a little bit of missing text prompting the user to set the collection concept ID to a collection they have the correct privileges for.

Maybe there should be a cell where the collection concept ID is set early on (kind of like the `Authorization` header is now) and then that same variable is called in all the later cells? Also, I think we only need one collection here, for simplicity.

Reply:
* [ ] Create cell for setting collection concept id

Reply:
Can we actual use two example datasets? I think it's clearer if we have one that we use `generate_collections_umm_var` for and another we use all smaller functions for

Reply:
Yeah those were my thoughts!